src/components/custom/_signup-affiliate-details.js

```diff
@@ -309,35 +295,43 @@ const SignupAffiliateDetails = ({ autofocus, handleLogin }) => {
                             </InputGroup>
                             <Flex fd="row" jc="space-between" ai="center" mt="2.4">
                                 <Line />
-                                <StyledText color="grey-5" align="center" tabletFontSize="12px">
-                                    {localize('Choose a plan')}
-                                </StyledText>
-                                <Tooltip
-                                    title={
-                                        <div style={{ color: 'black' }}>
-                                            {localize(
-                                                'Choose which plan you would like to subscribe to.',
-                                            )}{' '}
-                                            <LocalizedLinkText
-                                                to="/partners/affiliate-ib#deriv-ap"
-                                                external="true"
-                                                color="red"
-                                                target="_blank"
-                                                is_anchor
-                                                style={{ fontSize: '14px' }}
-                                            >
-                                                {localize('Learn more')}
-                                            </LocalizedLinkText>
-                                        </div>
-                                    }
-                                    color={'#d6dadb'}
-                                >
-                                    <img
-                                        style={{ marginLeft: '5px', cursor: 'pointer' }}
-                                        src={Info}
-                                        alt="info"
-                                    />
-                                </Tooltip>
+                                <Flex fd="row" ai="center" width="40%">
+                                    <Header
+                                        type="paragraph-2"
```

Comment:
Please include `as` prop for all Headers.

Reply:
Checked every `Header` on `as` property presence.